Voyager operator continue crash forbidden permission

Hi!

i continue to see this error on voyage-operator
```
[ffb45024-4d85-4db0-b56c-2841b8d293e0] Deleting Stats Service xxx/voyager-xxx-xxxx-stats
panic: ingresses.extensions "xxx" is forbidden: User "system:serviceaccount:kube-system:voyager-operator" cannot update resource "ingresses/status" in API group "extensions" in the namespace "xxx"
```

Voyage version is the latest: v12.0.0-rc1, i've checked the rbac permission and i have it. 

I've a GKE cluster with private worker node, i use helm on terraform to install and i set the value rbac.create = true.

How could i fix that?

--- a/charts/voyager/templates/cluster-role.yaml
+++ b/charts/voyager/templates/cluster-role.yaml
@@ -43,6 +43,7 @@ rules:
   - extensions
   resources:
   - ingresses
+  - ingresses/status
   verbs: ["*"]
 - apiGroups: [""]
   resources:

